requirements.txt

```diff
@@ -1,3 +1,7 @@
 flask
```

Comment:
Absence of versions tells `pip` to pick up the latest one, it leads to inconsistency. Please add specific versions to the packages.